tests/Integration/FrameworkIntegrationTestCase.php

```diff
@@ -28,6 +29,7 @@ protected function setUp(): void
     {
         $this->appConfig = new AppConfig(
             root: __DIR__ . '/../../',
+            environment: Environment::TESTING,
```

Comment:
was set to local before, but in here we're testing. so set the environment accordingly